utils/ixdtf/src/parsers/mod.rs

```diff
@@ -76,8 +77,8 @@ pub struct IxdtfParser<'a> {
 }
 
 impl<'a> IxdtfParser<'a> {
-    /// Creates a new `IXDTFParser` from a provided `&str`.
-    pub fn new(value: &'a str) -> Self {
+    /// Creates a new `IxdtfParser` from a provided `&str`.
+    pub fn new(value: &'a [u8]) -> Self {
```

Comment:
Given the talk during the call today. Was there any preference here around using `new` vs. `from_bytes`?

Reply:
Indeed; `from_utf8` might be the correct name here.

You can also add `from_str` (concrete associated function, not FromStr impl) and use it in the docs so you don't need to call `.as_bytes()` everywhere.